worker/migrations/migrate-accounts-to-organisation.go

```diff
@@ -98,6 +98,22 @@ func getCipherLinkFromManifest(manifest *app.KonnManifest) (string, error) {
     return link, nil
 }
 
+func linkAccountToCipher(acc * account.Account, cipher * bitwarden.Cipher) {
```

Comment:
nit: I would have avoided the "link" terminology as it has a different semantic than "relationship" in json-api. Maybe `addCipherRelationshipToCipher`?

Reply:
:+1:. I used addCipherRelationshipToAccount.